pkg/network/protocols/events/consumer.go

```diff
@@ -91,6 +92,7 @@ func NewConsumer[V any](proto string, ebpf *manager.Manager, callback func([]V))
 	eventsCount := metricGroup.NewCounter("events_captured")
 	kernelDropsCount := metricGroup.NewCounter("kernel_dropped_events")
 	invalidEventsCount := metricGroup.NewCounter("invalid_events")
```

Comment:
Yes, I have also changed it but haven’t pushed it yet because I’m validating the change. Additionally, I’m adding tags to the metric to pinpoint the exact root cause. I’ll push it soon! :)